certified-connectors/PPM Express/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2953 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/Webhooks/{id}": {
+      "get": {
+        "tags": [
+          "Webhooks"
+        ],
+        "summary": "Get a Webhook",
+        "description": "Gets a Webhook",
+        "operationId": "Webhooks_Get",
+        "x-ms-visibility": "internal",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "Webhook Id",
+            "x-ms-url-encoding": "single",
+            "x-ms-summary": "Webhook Id"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns existing item",
+            "schema": {
+              "$ref": "#/definitions/WebhookResponseResponse"
+            },
+            "examples": {
+              "application/json": {
+                "data": {
+                  "id": "d021c105-fc11-4cec-b45d-d6eba05ef410",
+                  "eventTypes": [
+                    "Project:Created",
+                    "Project:Updated"
+                  ],
+                  "headers": [
+                    {
+                      "name": "Authorization",
+                      "value": "Bearer token_value"
+                    },
+                    {
+                      "name": "custom_header",
+                      "value": "header_value"
+                    }
+                  ],
+                  "httpVerb": "POST",
+                  "url": "https://example.com"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "If not authorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "If the item is not found",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Webhooks"
+        ],
+        "summary": "Delete a Webhook",
+        "x-ms-visibility": "internal",
+        "description": "Deletes a Webhook",
+        "operationId": "Webhooks_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-ms-summary": "Webhook Id",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {          
+          "200": {
+            "description": "Ok",
+            "schema": {
+              "type": "object"
+            },
+            "headers": {}

```

Comment:
@vsorokin-ppmexpress should this one be empty? Are you going to use this property?